tests/components/ozw/test_lock.py

```diff
@@ -39,3 +39,58 @@ async def test_lock(hass, lock_data, sent_messages, lock_msg):
     msg = sent_messages[1]
     assert msg["topic"] == "OpenZWave/1/command/setvalue/"
     assert msg["payload"] == {"Value": False, "ValueIDKey": 173572112}
+
+    # Test set_usercode
+    await hass.services.async_call(
+        "ozw",
+        "set_usercode",
+        {
+            "entity_id": "lock.danalock_v3_btze_locked",
+            "usercode": 123456,
+            "code_slot": 1,
+        },
+        blocking=True,
+    )
+    assert len(sent_messages) == 3
+    msg = sent_messages[2]
+    assert msg["topic"] == "OpenZWave/1/command/setvalue/"
+    assert msg["payload"] == {"Value": "123456", "ValueIDKey": 281475150299159}
+
+    # Test get_usercode
+    await hass.services.async_call(
+        "ozw",
+        "get_usercode",
+        {"entity_id": "lock.danalock_v3_btze_locked", "code_slot": 1},
+        blocking=True,
+    )
+    assert len(sent_messages) == 3
+    assert "User code at slot 1 is:" in caplog.text
```

Comment:
I wasn't able to figure out how to get the test to pull this information since it's looking in the command class, or even if it was a relevant test.